pubspec.lock

```diff
@@ -5,154 +5,154 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      url: "https://pub.dartlang.org"
+      url: "https://pub.flutter-io.cn"
```

Comment:
Please remove this file from your commit since it changes all the urls to Chinese ones.